cloudinit/stages.py

```diff
@@ -757,6 +757,12 @@ def _consume_vendordata(self, vendor_source, frequency=PER_INSTANCE):
             LOG.debug("%s consumption is disabled.", vendor_source)
             return
 
+        if isinstance(enabled, str):
+            LOG.debug(
+                "Use of string for 'vendor_data:enabled' field is "
```

Comment:
Let's add the value here for debug/triage purposes.
```suggestion
                f"Use of string '{enabled}' for 'vendor_data:enabled' field is "
```